thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/components/ThresholdSeverityLabeler.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.pinot.thirdeye.detection.components;
+
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+import org.apache.pinot.thirdeye.anomaly.AnomalySeverity;
+import org.apache.pinot.thirdeye.datalayer.dto.MergedAnomalyResultDTO;
+import org.apache.pinot.thirdeye.detection.InputDataFetcher;
+import org.apache.pinot.thirdeye.detection.annotation.Components;
+import org.apache.pinot.thirdeye.detection.annotation.DetectionTag;
+import org.apache.pinot.thirdeye.detection.spec.SeverityThresholdLabelerSpec;
+import org.apache.pinot.thirdeye.detection.spi.components.Labeler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.apache.pinot.thirdeye.detection.spec.SeverityThresholdLabelerSpec.Threshold;
+
+/**
+ * Threshold-based severity labeler, which labels anomalies with severity based on deviation from baseline and duration
+ * of the anomalies. It tries to label anomalies from highest to lowest if deviation or duration exceeds the threshold
+ */
+@Components(title = "ThresholdSeverityLabeler", type = "threshold_severity_labeler",
```

Comment:
let's make the type field follow the existing convention, like `THRESHOLD_SEVERITY_LABELER`